include/SFML/Graphics/Glsl.inl

```diff
@@ -106,7 +106,10 @@ struct Vector4
     ////////////////////////////////////////////////////////////
     /// \brief Construct from 4 vector components
     ///
-    /// \param X,Y,Z,W Components of the 4D vector
+    /// \param X Component of the 4D vector
+    /// \param Y Component of the 4D vector
+    /// \param Z Component of the 4D vector
+    /// \param W Component of the 4D vector
```

Comment:
> clang bugreport!

Technically, I would need to go through Apple's bug report system as they modify clang for their needs... But you're right -- and I totally agree on the readability/conciseness part. However, last time I used it, it took them two years to answer my bug report. In the meantime, let's have a happy BuildBot report, shall we? :smile: